content/200-concepts/100-components/01-prisma-schema/06-relations.mdx

```diff
@@ -9,17 +9,18 @@ tocDepth: 3
 
 A relation is a _connection_ between two models in the Prisma schema. For example, there is a one-to-many relation between `User` and `Post` because one user can have many blog posts. The following Prisma schema defines a one-to-many relation between the `User` and `Post` models. The fields involved in defining the relation are highlighted:
 
-```prisma line-number file=schema.prisma
+```prisma line-number file=schema.prisma 
 model User {
-  id      Int      @id @default(autoincrement())
-|  posts   Post[]
+  id     Int     @id @default(autoincrement())
+|  posts  Post[]
```

Comment:
This still looks off when comparing alignment with the above row

Reply:
Not when looking at the rendered version, where the `|` is turned into a highlight of that line.

Reply:
Ah, fair play. I didnt pull this in and check it our in the browser